Loop/Luminare/Settings/Keybindings/KeybindingItem.swift

```diff
@@ -163,21 +159,26 @@ struct KeybindingItemView: View {
     }
 
     func label() -> some View {
-        HStack(spacing: 0) {
-            HStack(spacing: 8) {
-                IconView(action: $keybind)
-
-                Text(keybind.getName())
-                    .lineLimit(1)
-                    .contentTransition(.numericText())
-                    .animation(LuminareSettingsWindow.animation, value: keybind)
-            }
+        Button {
+            isPresented.toggle()
+        } label: {
+            HStack(spacing: 0) {
+                HStack(spacing: 8) {
+                    IconView(action: $keybind)
+
+                    Text(keybind.getName())
+                        .lineLimit(1)
+                        .contentTransition(.numericText())
+                }
 
-            if let info = keybind.direction.infoView {
-                info
+                if let info = keybind.direction.infoView {
+                    info
+                }
             }
+            .fixedSize(horizontal: false, vertical: true)
         }
-        .fixedSize(horizontal: false, vertical: true)
+        .buttonStyle(CompactButtonStyle())
+        .help("Customize this keybind's action.")
```

Comment:
Is this possible to be translated? I forget about `.help()` values.

Reply:
Yes! We've been using `.help()` values since 1.0.0, so they are already localized in most languages too!